### ISSUE TITLE
Fix `epix_slide` example referring to nonexistent column

### ISSUE THREAD

**Issue by brookslogan** [cmu-delphi/epiprocess#137]
```
options(warn=1L)
devtools::run_examples()
```
produces (along with some other output and warnings)
```
> epix_slide(x = archive_cases_dv_subset,
+            f = ~ mean(.x$case_rate),
+            n = 3,
+            group_by = geo_value,
+            ref_time_values = time_values,
+            new_col_name = 'case_rate_3d_av')
Warning: Unknown or uninitialised column: `case_rate`.
Warning in mean.default(.x$case_rate) :
  argument is not numeric or logical: returning NA
Warning: Unknown or uninitialised column: `case_rate`.
Warning in mean.default(.x$case_rate) :
  argument is not numeric or logical: returning NA
Warning: Unknown or uninitialised column: `case_rate`.
Warning in mean.default(.x$case_rate) :
  argument is not numeric or logical: returning NA
[....................]
```
It looks like the column has been renamed or removed (maybe while addressing #83):
```
> archive_cases_dv_subset[["DT"]] %>% names()
[1] "geo_value"       "time_value"      "version"         "percent_cli"    
[5] "case_rate_7d_av"
```
So we should either change the column used for this computation, or reintroduce the old column and address #83 via selecting the interesting columns somewhere within those examples.

**Comment by rachlobay** [comment]
I could be wrong, but I think this relates back to changes made a couple months ago to the internal data (for ex. https://github.com/cmu-delphi/epiprocess/commit/d8cf12485622e95038ec8c56197af6eba3004658) and so looking at those changes may or may not help here.

**Comment by mgyliu** [comment]
Yeah I traced it down to the same source. Thanks for commenting @rachlobay!

Summary of why this happened: When `archive_cases_dv_subset` was first created ([this PR](https://github.com/cmu-delphi/epiprocess/commit/f92f55c163eabf3d33aaeafa4f2a0771c001d79c#diff-d56b40ce7e1add91bed8f046498766df71beb822ef090f8a71dd68f0ff09ecd8R29)), it was called `archive_cases_dv` and the column was called `case_rate`. In a [subsequent PR](https://github.com/cmu-delphi/epiprocess/commit/2df3625c92c833486466e6cdffad50acf6c3f16a#diff-4d2c46312f29e0e03d0573478a6e124f41ff07b9b8fb188424895f7af4a49748L29-R29), the dataset was renamed to `archive_cases_dv_subset` and the column was changed to `case_rate_7d_av`. The column values themselves were the same across both PRs (they were always the 7-day average case rate).

To fix the warnings, I changed the column in the example to `case_rate_7d_av`.

Currently having trouble with `devtools::check` on my local machine but I will put up a PR for this soon!
